test/pkg/environment/common/expectations.go

```diff
@@ -222,30 +194,6 @@ func (env *Environment) ExpectConfigMapDataOverridden(key types.NamespacedName,
 	return true
 }
 
-func (env *Environment) ExpectPodENIEnabled() {
```

Comment:
Assuming this is also just general cleanup too. Would prefer it to be moved along with any other general cleanup to a separate PR.

Reply:
https://github.com/Azure/karpenter-provider-azure/pull/240